hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueService.java

```diff
@@ -430,6 +430,10 @@ public void provideDynamicMetrics(MetricDescriptor descriptor, MetricsCollection
         provide(descriptor, context, QUEUE_PREFIX, getStats());
     }
 
+    public void resetAgeStats(String queueName) {
```

Comment:
This would throw a NPE if the queue is not created yet.

Reply:
Fixed, thanks.